smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/httpResponse/HttpResponseGenerator.kt

```diff
@@ -36,7 +36,6 @@ class HttpResponseGenerator(
         httpOperations.forEach {
             httpResponseBindingErrorGenerator.render(ctx, it)
             HttpResponseBindingErrorNarrowGenerator(ctx, it, unknownServiceErrorSymbol).render()
-            WaiterTypedErrorGenerator(ctx, it, unknownServiceErrorSymbol).render()
```

Comment:
Removes `WaiterTypedError` conformance from every operation error.  Now `WaiterTypedError` conformance is only added where actually needed.